### ISSUE TITLE
Like config file, have a results file?

### ISSUE THREAD

**Issue by Ditiae** [NightShadeI/Revolution-Rotation-Calculator-#20]
Title.
oh also, dragon rider amulet support? or nah

**Comment by NightShadeI** [comment]
I will see if i can work on this, busy right now so I'll see what I can do on the weekend. As for dragon rider amulet will likely incorporate this with lots of other items/ perks.

**Comment by EZtouch** [comment]
I think logging the results in a file is fine, but we should still print it in the command line window.

**Comment by Ditiae** [comment]
Yes, we should print in command window. But logging a results file will mean if people accidently close the window or need to rerun it again sometime, they already have the result.
